applications/asset_tracker_v2/src/modules/Kconfig.data_module

```diff
@@ -63,6 +63,11 @@ config DATA_ACCELEROMETER_BUFFER_COUNT
 	range 1 100
 	default 3
 
+config DATA_IMPACT_BUFFER_COUNT
+	int "Number of impact data ringbuffer entries"
+	range 1 100
+	default 3
```

Comment:
Maybe we should just set this buffer to 1 and let the customer decide if its appropriate to increase the buffer in their use case.